src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/event/CitizenCreateApplication.java

```diff
@@ -91,9 +91,20 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
             temp.substring(8, 12),
             temp.substring(12, 16)
         ));
+        setDssMetaData(data);
 
         return AboutToStartOrSubmitResponse.<CaseData, State>builder()
             .data(data)
             .build();
     }
+
+    private void setDssMetaData(CaseData data) {
+
+        data.setDssQuestion1("Full Name");
+        data.setDssQuestion2("Date of Birth");
+        data.setDssQuestion3("Test");
+        data.setDssAnswer1("case_data.childrenFirstName, case_data.childrenLastName");
```

Comment:
If the backend has different fields for First name and last name and i believe, its been asked the same way while the case is created. Then DSS question should also ask for first name and last name separately. As if they are asked combined, we dont know if the data entered is Firstname first or last name first.

@YasharRahvar , correct me if i am wrong.

Reply:
@yogendraU Standard way of entering full name is always first name followed by last name. If user enter last name first, then it is an incorrect answer. It's also an option for adoption team to make it clear to the user that they need to enter first name followed by last name in the question.
for example: What is applicant full name? (enter first name followed by last name)

Reply:
@sairamkantheti1 , could you please segregate first name and last name as two different questions

Reply:
@yogendraU  Sure. I will make this change. Please update the same in comments of below user story including the exact question label
https://tools.hmcts.net/jira/browse/ADOP-1973

Reply:
@sairamkantheti1 , i have updated the jira with comment

Reply:
@yogendraU made changes as per comments